airlock/business_logic.py

```diff
@@ -1769,6 +1769,31 @@ def update_file_in_request(
         release_request: ReleaseRequest,
         relpath: UrlPath,
         user: User,
+    ) -> ReleaseRequest:
+        request_file = release_request.get_request_file_from_output_path(relpath)
+        return self._delete_from_then_add_file_to_request(
```

Comment:
I think it makes sense to re-use this code, but I don't like the name of the method. `replace_file_in_request`?  Or maybe it just needs some comments to explain why we have to delete and then re-add a file.

Reply:
`replace_file_in_request` is a much better name!